### ISSUE TITLE
Disable probing device mapper

### ISSUE THREAD

**Issue by sviperz** [firehol/netdata#3974]
Trying to clean up errors from the log. The only errors I wasn't able to suppress is a DEVICE-MAPPER related ones. During the start there are many errors in the log like that:
```
DEVICE-MAPPER ('mtdblock0', 31:0): Cannot open directory '/dev/mapper'. Disabling device-mapper support.
DEVICE-MAPPER ('mtdblock0', 31:0): Cannot open directory '/dev/disk/by-label'. Disabling device-mapper support.
```
I've tried to blank _path to device mapper_ in _netdata.conf_, but it doesn't work. Is there any parameter I can use which will disable device mapper probing completely?

**Comment by ktsaou** [comment]
I think you can set these directories to nothing, like this:

```
[plugin:proc:/proc/diskstats]
	path to device mapper = 
	path to /dev/disk/by-label = 
```

**Comment by sviperz** [comment]
That's exactly what I have already tried, but it doesn't suppress any device mapper error messages from the log. Actually, setting these directories to nothing doesn't make any difference at all to the log output.

**Comment by ktsaou** [comment]
You are right. The code did not allow empty values in `netdata.conf` settings, even if the rest of the code could handle the case.
Fixed it in PR #4208

**Comment by ktsaou** [comment]
merged it.
